src/classes/RD2_VisualizeScheduleController_TEST.cls

```diff
@@ -776,7 +776,78 @@ private with sharing class RD2_VisualizeScheduleController_TEST {
         System.assertNotEquals(null, record.dayOfMonth, 'DayOfMonth should be specified');
     }
 
+    /****
+    * @description Validates that a active schedule is returned for when RD Status is mapped to Active
+    */
+    @IsTest
+    private static void shouldReturnActiveScheduleWhenRDStatusIsMappedToActive() {
+        Date startDate = Date.newInstance(2018, 2, 1);
+        Date today = Date.newInstance(2019, 7, 31);
+        String dayOfMonth = '12';
+
+        RD2_EnablementService_TEST.setRecurringDonations2Enabled();
+
+        npe03__Recurring_Donation__c rd = getRecurringDonationMonthlyBuilder()
+            .withStartDate(startDate)
+            .withDayOfMonth(dayOfMonth)
+            .withStatusClosed()
+            .build();
 
+        RD2_StatusMapper_TEST.Stub mapperStub = new RD2_StatusMapper_TEST.Stub()
+            .withStatusMapping(RD2_Constants.STATUS_CLOSED, RD2_Constants.STATUS_ACTIVE);
```

Comment:
Let's use a new status value so it can be mapped on an Active state, instead of using the read-only mapping of Closed status to Closed state. The same comment is applicable to the next test.